src/sickle.ts

```diff
@@ -66,9 +66,31 @@ class Annotator {
         this.writeNode(classNode.getLastToken());
         break;
       }
+      case ts.SyntaxKind.PublicKeyword:
```

Comment:
agree, we can make sickle throw in that case. But if we do nothing, someone will write it sooner or later and sickle will produce invalid closure code.